### PR TITLE
Some build script cleanup

### PR DESCRIPTION
**Pull request by jrodbx**

### CODE REVIEW

File: paparazzi/build.gradle [hunk 1]
```diff
@@ -8,11 +7,6 @@ apply plugin: 'com.vanniktech.maven.publish'
 apply plugin: 'dev.drewhamilton.poko'
 apply plugin: 'aar2jar'
 
-java {
-  sourceCompatibility = libs.versions.javaTarget.get()
-  targetCompatibility = libs.versions.javaTarget.get()
-}
-
```
Comment (jrodbx) [inline paparazzi/build.gradle:15]: duplicate; already handled in root script

File: paparazzi/build.gradle [hunk 2]
```diff
@@ -149,14 +143,4 @@ tasks.withType(Test).configureEach {
...
-  // Uncomment to debug JNI issues in layoutlib
-  // jvmArgs '-Xcheck:jni'
-  testLogging {
-    events 'passed', 'failed', 'skipped'
-    exceptionFormat 'FULL'
-    showCauses true
-    showExceptions true
-    showStackTraces true
-  }
```
Comment (jrodbx) [inline paparazzi/build.gradle:161]: moved to root script

File: settings.gradle [hunk 3]
```diff
@@ -1,10 +1,5 @@
 rootProject.name = 'paparazzi-root'
 
-include ':libs:layoutlib'
-include ':libs:native-macarm'
-include ':libs:native-macosx'
-include ':libs:native-win'
-include ':libs:native-linux'
```
Comment (jrodbx) [inline settings.gradle:7]: obsoleted by migration to Google's layoutlib

File: build.gradle [hunk 4]
```diff
@@ -90,15 +90,6 @@ subprojects {
...
 
-  tasks.register('emptySourcesJar', Jar) {
-    // TODO: fetch sources from the corresponding AOSP repos.
-    archiveClassifier = 'sources'
-  }
-
-  tasks.register('emptyJavadocJar', Jar) {
-    archiveClassifier = 'javadoc'
-  }
-
```
Comment (jrodbx) [inline build.gradle:101]: obsoleted by migration to Google's layoutlib (we used the prebuilt binaries, and therefore had no sources/javadocs artifacts).

File: build.gradle [hunk 5]
```diff
@@ -41,6 +39,8 @@ subprojects {
     //mavenLocal()
   }
 
+  // Uncomment to debug JNI issues in layoutlib
+  // jvmArgs '-Xcheck:jni'
```
Comment (jrodbx) [inline build.gradle:43]: relates to https://github.com/cashapp/paparazzi/pull/1458/files#r1619310098

File: build.gradle [hunk 6]
```diff
@@ -29,7 +27,7 @@ buildscript {
 apply plugin: 'binary-compatibility-validator'
 
 apiValidation {
-  ignoredProjects += ['libs', 'sample']
+  ignoredProjects += ['sample']
```
Comment (jrodbx) [inline build.gradle:30]: relates to https://github.com/cashapp/paparazzi/pull/1458/files#r1619310413

File: build.gradle [hunk 7]
```diff
@@ -1,5 +1,3 @@
-import org.jetbrains.kotlin.gradle.dsl.KotlinTopLevelExtension
```
Comment (jrodbx) [inline build.gradle:1]: unused import

File: paparazzi/build.gradle [hunk 8]
```diff
@@ -1,4 +1,3 @@
-import org.jetbrains.kotlin.gradle.plugin.AbstractKotlinPluginKt
```
Comment (jrodbx) [inline paparazzi/build.gradle:1]: unused import

**Review by JakeWharton — APPROVED** [review]

**Pull request merged**